nuget/lib/dependabot/nuget/nuget_config_credential_helpers.rb

```diff
@@ -48,6 +50,36 @@ def self.add_credentials_to_nuget_config(credentials)
         File.write(user_nuget_config_path, nuget_config)
       end
 
+      sig { params(credentials: T::Array[Dependabot::Credential]).returns(T::Hash[String, String]) }
+      def self.get_credentials_to_credential_helper_env(credentials)
+        env = {}
+
+        nuget_credentials = credentials.select { |cred| cred["type"] == "nuget_feed" }
+
+        if nuget_credentials.any?
+          endpoint_credentials = []
+
+          nuget_credentials.each do |c|
+            next unless c["token"]
```

Comment:
Azure DevOps feeds aren't necessarily private and authentication isn't always specified via a `token` field, it could also come in via `username`/`password`, so that'll have to be accounted for.